cmake/sysbuild/fast_pair_hex.cmake

```diff
@@ -58,31 +70,44 @@ function(fast_pair_hex_dts)
   set(fp_partition_name bt_fast_pair_partition)
 
   sysbuild_dt_nodelabel(
-    bt_fast_pair_partition_nodelabel
+    bt_fast_pair_partition_node_full_path
     IMAGE
     ${DEFAULT_IMAGE}
     NODELABEL
     "${fp_partition_name}"
     )
+
   sysbuild_dt_reg_addr(
     bt_fast_pair_partition_relative_address
     IMAGE
     ${DEFAULT_IMAGE}
     PATH
-    "${bt_fast_pair_partition_nodelabel}"
+    "${bt_fast_pair_partition_node_full_path}"
     )
 
-  # This part assumes that the Fast Pair partition resides in MRAM1x.
-  # TODO: Rewrite it to be more generic and support other memory regions.
-  sysbuild_dt_nodelabel(mram1x_nodelabel IMAGE ${DEFAULT_IMAGE} NODELABEL "mram1x")
-  sysbuild_dt_reg_addr(mram1x_address IMAGE ${DEFAULT_IMAGE} PATH "${mram1x_nodelabel}")
+  dt_get_parent(
```

Comment:
Perhaps a comment regarding the assumed DTS structure for the partition node would be helpful here (possibly with a link to Zephyr documentation).

Reply:
thank you